core/templates/dev/head/admin/admin.html

```diff
@@ -23,7 +23,7 @@
       <div class="navbar-container" style="background-color: #00376d">
         <div class="navbar-header protractor-test-navbar-header pull-left">
           <a class="oppia-navbar-brand-name oppia-transition-200" href="/library">
-            <img src="/images/logo/288x128_logo_white.png" class="oppia-logo" ng-class="'oppia-logo-wide'">
+            <img src="{{get_static_resource_url('/assets/images/logo/288x128_logo_white.png')}}" class="oppia-logo" ng-class="'oppia-logo-wide'">
```

Comment:
Yep, it is page-level, but following our discussion let's try to not use jinja as much as possible so that we can start caching the html templates.